components/org.wso2.carbon.identity.api.user.totp/org.wso2.carbon.identity.api.user.totp.common/src/main/java/org/wso2/carbon/identity/api/user/totp/common/TOTPConstants.java

```diff
@@ -23,6 +23,8 @@
 
     public static final String USER_TOTP_PREFIX = "OTP-";
 
+    public static final String AUTHENTICATED_WITH_BASIC_AUTH = "AuthenticatedWithBasicAuth";
```

Comment:
Can't we declare this in a single place?

Reply:
This will be covered in the next totp improvement.